tools/topology/development/sof-tgl-nocodec-ci.m4

```diff
@@ -230,7 +230,7 @@ DAI_ADD(sof/pipe-dai-capture.m4,
 # capture DAI is DMIC 0 using 2 periods
```

Comment:
Minor, the comment text should be DMIC 1.

Reply:
> Minor, the comment text should be DMIC 1.

Thanks, fixed now.